tests/storagenym/CMakeLists.txt

```diff
@@ -0,0 +1,19 @@
+set(name unittests-opentxs-storagenym)
+
+set(cxx-sources
+        main.cpp
+        Test_StorageNym.cpp
+        )
+
+include_directories(
```

Comment:
This statement is unnecessary and causes build errors because not all of these directories exist.